Add page with list of extensions (#1115)
linkerd/website: linkerd.io/content/2.10/reference/extension-list.md

+++ 
title = "Extensions List" 
description = "List of Linkerd extensions that can be added to the installation for additional functionality"
+++

Linkerd provides a mix of built-in and third-party
[extensions](../../tasks/extensions/) to add additional functionality to the
base installation. The following is the list of known extensions:

{{< extensions-2-10 >}}

> If you have an extension for Linkerd and it is not on the list, [please submit
> a pull request!](https://github.com/linkerd/website)

Reviewer: May I suggest a more direct link to edit the list?
```suggestion
If you have an extension for Linkerd and it is not on the list, [please edit
this
page!](https://github.com/linkerd/website/edit/main/linkerd.io/data/extension-list.yaml)
```
